libs/designsystem/src/lib/components/highstock-chart/options/highstock-chart-options.ts

```diff
@@ -0,0 +1,221 @@
+import { formatNumber } from '@angular/common';
+
+import { Options } from 'highcharts';
+import * as Highcharts from 'highcharts/highstock';
```

Comment:
Consider only importing what's used:
```suggestion
import { dateFormat } from 'highcharts/highstock';
```

Reply:
Done.

Reply:
Done.